Hanford/Cloud/AzureSQL/Create Hanford tables.sql

```diff
@@ -52,6 +58,7 @@ CREATE TABLE [dbo].[Messages](
 	[DeviceId]        [nvarchar](50)     NOT NULL,
 	[DeviceTimestamp] [datetime]         NOT NULL,
 	[MessageType]     [smallint]         NOT NULL,
+	[UtcStamp]        [datetime]         NOT NULL,
```

Comment:
Why add this when you already have DeviceTimestamp (which is really IoTHub enqueueTimestamp)?